scripts/run.js

```diff
@@ -7,27 +7,36 @@ const chalk = require('chalk');
 const concurrently = require('concurrently');
 
 const {makeRunner} = require('./runner.js');
-const {getWorkspaceCommands} = require('./utils.js');
+const {getProductStartCommands, getWorkspaceCommands} = require('./utils.js');
 
 async function watchAll(useRunner) {
-    const commands = [];
+    if (!useRunner) {
```

Comment:
What does using a runner mean?

Reply:
That's the toggle for the slightly fancier terminal UI I added for those commands with tabs for filtering. It has some quirks and things that don't work right, so I made it optional. It also only works with `make run` and not `make dev` although I could probably make it work for both
<img width="489" alt="Screen Shot 2023-01-20 at 11 58 21 AM" src="https://user-images.githubusercontent.com/3277310/213758223-57d2a70f-a39f-4e77-a89e-520387cc6459.png">